services/source-debugger/eventUploader.go

```diff
@@ -24,16 +24,16 @@ type GatewayEventBatchT struct {
 
 //EventUploadT is a structure to hold actual event data
 type EventUploadT struct {
-	Event             string      `json:"event,omitempty"`
+	Event             interface{} `json:"event,omitempty"`
```

Comment:
@psrikanth88 As per our discussion in the channel, let's leave everything as string as earlier. 
We will typecase the UserID field to string in the gateway.